client/client-multi/client-c/toolchains/cc32xx.cmake

```diff
@@ -16,21 +16,78 @@
 
 include(CMakeForceCompiler)
 
-if(DEFINED WIN32)
-  set(EXECUTABLE_SUFFIX .exe)
-else()
-  set(EXECUTABLE_SUFFIX)
-endif()
-
-if(NOT DEFINED KAA_TOOLCHAIN_PATH)
-  set(KAA_ARM_TOOLCHAIN "/opt/kaa/gcc-arm-none-eabi")
+set(CMAKE_SYSTEM_NAME Generic)
+
+set(ARM_GCC_COMPILER "arm-none-eabi-gcc${CMAKE_EXECUTABLE_SUFFIX}")
+
+# Find toolchain path
+
+if (NOT DEFINED ENV{CC32XX_TOOLCHAIN_PATH})
+  # Check if GCC is reachable.
+  find_path(TOOLCHAIN_PATH bin/${ARM_GCC_COMPILER})
```

Comment:
Original idea was to use TOOLCHAIN_PATH pointing to dir containing whole toolchain, not only binaries:
[https://github.com/kaaproject/sample-apps/blob/master/configurationdemo/source/cc32xx/platform/cc32xx/cc32xx.cmake](source)

But I'm fine to change it, this will not break anything I hope.